static/js/browser/browser_page.tsx

```diff
@@ -176,31 +176,21 @@ export class BrowserPage extends React.Component<
       : this.props.dcid;
   }
 
-  private isProvenanceEntity(prov): boolean {
-    return (
-      prov["subjectTypes"] &&
-      prov["subjectTypes"][0] === "Provenance" &&
-      !!prov["subjectName"]
-    );
-  }
-
   private fetchData(): void {
     const provenancePromise = axios
-      .get("/api/browser/triples/Provenance")
+      .get("/api/browser/provenance")
       .then((resp) => resp.data);
     const labelsPromise = axios
       .get("/api/browser/proplabels/" + this.getArcDcid())
       .then((resp) => resp.data);
     Promise.all([labelsPromise, provenancePromise])
       .then(([labelsData, ProvenanceData]) => {
         const provDomain = {};
-        for (const prov of ProvenanceData) {
-          if (this.isProvenanceEntity(prov)) {
-            try {
-              provDomain[prov["subjectId"]] = new URL(prov["subjectName"]).host;
-            } catch (err) {
-              console.log("Invalid url in prov: " + prov["subjectName"]);
-            }
+        for (const prov in ProvenanceData) {
```

Comment:
nit: while we're here, can we fix the var casing? s/ProvenanceData/provenanceData/

Reply:
Done